packages/devtools_app/lib/src/service/editor/api_classes.dart

```diff
@@ -249,6 +257,27 @@ class DebugSessionStoppedEvent extends EditorEvent {
       };
 }
 
+class ThemeChangedEvent extends EditorEvent {
+  ThemeChangedEvent({required this.theme});
+
+  ThemeChangedEvent.fromJson(Map<String, Object?> map)
```

Comment:
can this be a factory constructor instead?

Reply:
Leaving as named constructor